packages/compile/src/model/read-equations.js

```diff
@@ -114,6 +114,9 @@ class Context {
     const parsedModel = { kind: 'vensim', root: parseVensimModel(eqnText) }
 
     // Create one or more `Variable` instances from the equations
+    // TODO: Technically we should pass `spec.separateAllVarsWithDims` here so that any
```

Comment:
I added a TODO here because it's possible that you could end up with generated vars that are not separated as expected.  In practice I don't think this is likely to happen given that `defineVariables` is typically used in cases where the variables are already non-apply-to-all, but I could be wrong about that.  Anyway, I think it's fine to not act on this now but wanted to flag it in case you think we should fix it now (it wouldn't be too hard to pass the `spec` to this function).